packages/systemd-252/systemd-252.spec

```
@@ -164,11 +170,40 @@ Provides: %{_cross_os}systemd-networkd = %{package_priority_epoch}:
 %package resolved
 Summary: Files for resolved
 Requires: %{name}
+Requires: %{name}-resolved(dbus-config)
 Provides: %{_cross_os}systemd-resolved = %{package_priority_epoch}:
 
 %description resolved
 %{summary}.
 
+%package dbus-launcher-config
+Summary: Dbus launcher configuration files
+Requires: %{name}
+Provides: %{name}(dbus-config) = 1:
+Provides: %{name}-resolved(dbus-config) = 1:
+Provides: %{name}-networkd(dbus-config) = 1:
```

Not sure there's much value in these separate capabilities if they're always provided as a group.
```suggestion
Provides: %{name}(dbus-config) = 1:
```